devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy

```diff
@@ -21,7 +21,10 @@ metadata {
 		capability "Alarm"
 		capability "Chime"
 
-		fingerprint mfr: "0371", prod: "0003", model: "00A2", deviceJoinName: "Aeotec Doorbell 6"
+		fingerprint mfr: "0371", prod: "0003", model: "00A2", deviceJoinName: "Aeotec Doorbell 6" //EU
+		fingerprint mfr: "0371", prod: "0103", model: "00A2", deviceJoinName: "Aeotec Doorbell 6" //US
+		fingerprint mfr: "0371", prod: "0003", model: "00A4", deviceJoinName: "Aeotec Siren 6" //EU
+		fingerprint mfr: "0371", prod: "0103", model: "00A4", deviceJoinName: "Aeotec Siren 6" //US
```

Comment:
Considering we have another DTH for the Gen 5 siren, are you sure you want to add this fingerprint here?

Reply:
I believe the Siren (Gen 5) and the Siren 6 have completely different integrations. The Siren 6 and Doorbell 6 are the same device (with the exception that the Doorbell 6 is shipped with a button).

Reply:
@dkirker yup, you're correct